### PR TITLE
Add Wifi connection debugging procedure. common-issues.md #3

### PR DESCRIPTION
**Pull request by Spazznyan**
Edited current "Tracker cant connect to WiFi" section for better wording

Added debugging procedure for usb serial to "Tracker cant connect to WiFi" section

Minor fixes in "Unable to find server" section

Adapted from my discord debugging text:
image

### CODE REVIEW

File: src/common-issues.md [hunk 1]
```diff
@@ -70,25 +70,47 @@ There are two common causes that you should check:
 
 ## Tracker can't connect to WiFi
 
-The two common issues that cause this error are:
+The most common issues that cause this error are:
 
 - Make sure you are connecting to a 2.4GHz network, 5GHz networks are not supported.
-- Check your SSID for special characters. At the time of writing SlimeVR only supports network SSIDs that contain alphanumerical characters.
+- Check your SSID for special characters and correct capitalisation. At the time of writing SlimeVR only supports network SSIDs that contain alphanumerical characters.
```
Comment (OstlerDev) [inline src/common-issues.md:76]: nit: fix spelling

Also, I tested out custom ssid names with unicode characters on v21.0.0 and esp 0.7.3, it worked just fine, so I think this changed recently at some point and they now work? It both worked in the "connect trackers" page and directly from the serial console without the need for using "SET BWIFI". I can double check this again, but it does seem to support ssids with unicode characters now. (didn't test passwords yet, will report back).

```suggestion
- Check your SSID for special characters and correct capitalization. At the time of writing SlimeVR only supports network SSIDs that contain alphanumerical characters.
```

File: src/common-issues.md [hunk 2]
```diff
@@ -70,25 +70,47 @@ There are two common causes that you should check:
...
 - Make sure you are using WiFi channels 1-11. Avoid using channels 12-14 because connection issues may occur.
 - Ensure WPA3 WiFi security is not being used, as the SlimeVR trackers do not support this security protocol. We recommend using WPA2, which is fully supported by SlimeVR.
 - Try restarting your router to see if this resolves the issue.
-- When using a WiFi 7 router, try disabling MLO or switching from 'Performance' to 'Compatibility' mode (sometimes called 'Max interoperability' mode).
+- Your router has reached the maximum number of connected devices. You can test this by disconnecting devices and then trying to connect your trackers again.
```
Comment (OstlerDev) [inline src/common-issues.md]: nit: clarify "other" devices so that people don't think it means to disconnect the trackers and reconnect them.

```suggestion
- Your router has reached the maximum number of connected devices. You can test this by disconnecting other devices from your wifi and then trying to connect your trackers again.
```

File: src/common-issues.md [hunk 3]
```diff
@@ -70,25 +70,47 @@ There are two common causes that you should check:
...
 
-If all of this is correct, you can check your gateway's list of connected devices to see if all your trackers are connecting. If a tracker is not connecting even after using the same firmware upload with hardcoded Wi-Fi details there are two additional steps you can check:
+### Debugging a tracker that is not connecting to WiFi:
+To find out more, press the "I'm having trouble connecting" button and check the serial logs of a plugged in tracker when it boots up. This will normally give u a good hint at where the issue is. You can also navigate there using settings > serial console.
```
Comment (OstlerDev) [inline src/common-issues.md]:
```suggestion
To find out more, press the "I'm having trouble connecting" button and check the serial logs of a plugged in tracker when it boots up. This will normally give you a good hint at what the issue is and will help others identify the problem you are having. You can also navigate there by going to Settings > Utilities > Serial console.
```

File: src/common-issues.md [hunk 4]
```diff
@@ -70,25 +70,47 @@ There are two common causes that you should check:
...
-- If you hard coded your Wi-Fi settings in `platformio.ini` try connecting your trackers via USB and [pushing new Wi-Fi details](server/connecting-trackers.md#connecting-trackers). You may find this either fixes your connection or provides you with additional details on why the connection is failing.
+**Press the "Reboot" button or switch the tracker off/on to see the following debug messages:**
+
+`[INFO] [WiFiHandler] Loaded credentials for SSID  and pass length 0`<br>
+↳ **Cause**: The WiFi credentials are not being written the tracker.<br>
+Try using the **SET WIFI "wifiname" "wifipassword"** command in the command box at the bottom of the page (e.g. **SET WIFI "SpazzWiFi" "Passw0rd1234"**). When successful, the serial log should display "WIFI SET OK".
+
+`[ERROR] [WiFiHandler] Can't connect from any credentials, error: _, reason: SSID not found.`<br>
+↳ **Cause**: The WiFi name that was input is not transmitting a 2.4ghz signal _or_ the name was misspelled.<br>
+Press the "Get WiFi Scan" button and check the WiFi name you entered against the results to ensure correct capitalisation and check for hidden spaces.
```
Comment (OstlerDev) [inline src/common-issues.md:93]:
```suggestion
Press the "Get WiFi Scan" button and check the WiFi name you entered against the results to ensure correct capitalization and check for hidden spaces. 
```

File: src/common-issues.md [hunk 5]
```diff
@@ -70,25 +70,47 @@ There are two common causes that you should check:
...
+
+`[ERROR] [WiFiHandler] Can't connect from any credentials, error: _, reason: SSID not found.`<br>
+↳ **Cause**: The WiFi name that was input is not transmitting a 2.4ghz signal _or_ the name was misspelled.<br>
+Press the "Get WiFi Scan" button and check the WiFi name you entered against the results to ensure correct capitalisation and check for hidden spaces. 
+
+`[ERROR] [WiFiHandler] Can't connect from any credentials, error: _, reason: incorrect password.`<br>
+↳ **Cause**: The password that was input is incorrect. Double check you have the correct password and try again.
+
+`[ERROR] [WiFiHandler] Can't connect from any credentials, error: _, reason: Timed out.`<br>
+↳ **Cause**: Your router is not responding to the trackers attempts at connection.<br>
```
Comment (OstlerDev) [inline src/common-issues.md:99]:
```suggestion
↳ **Cause**: Your router is not responding to the trackers attempts at connection, or your tracker is too far away from your router. <br>
```

**Comment by OstlerDev** [comment]
Looks great overall to me! There was a couple spelling errors and a couple places that some small clarifications might be helpful, but I think it should be good to go overall! Love the improvement.
- puppy sky

File: src/common-issues.md (hunk 4 above)
Comment (Spazznyan) [reply src/common-issues.md:93]: its S in british english

File: src/common-issues.md (hunk 1 above)
Comment (Spazznyan) [reply src/common-issues.md:76]: its S in british english

I believe only quotation marks (escape issue) and some really uncommon unicode wont work.
